[0m19:42:22.724646 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1064d5850>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10702e540>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10702e1b0>]}


============================== 19:42:22.726010 | bc8698dc-13f3-48cd-ad4c-2eabf9e83c68 ==============================
[0m19:42:22.726010 [info ] [MainThread]: Running with dbt=1.10.1
[0m19:42:22.726365 [debug] [MainThread]: running dbt with arguments {'log_cache_events': 'False', 'debug': 'False', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'quiet': 'False', 'cache_selected_only': 'False', 'invocation_command': 'dbt init pactravel_dbt_project', 'printer_width': '80', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': '/Users/macbookairm2/.dbt', 'log_path': 'logs', 'indirect_selection': 'eager', 'write_json': 'True', 'version_check': 'True', 'use_colors': 'True', 'log_format': 'default', 'use_experimental_parser': 'False', 'empty': 'None', 'introspect': 'True', 'partial_parse': 'True', 'target_path': 'None', 'send_anonymous_usage_stats': 'True', 'no_print': 'None'}
[0m19:42:22.731316 [info ] [MainThread]: Creating dbt configuration folder at /Users/macbookairm2/.dbt
[0m19:42:22.731964 [debug] [MainThread]: Starter project path: /opt/anaconda3/lib/python3.12/site-packages/dbt/include/starter_project
[0m19:42:22.734447 [info ] [MainThread]: 
Your new dbt project "pactravel_dbt_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m19:42:22.734657 [info ] [MainThread]: Setting up your profile.
[0m19:43:27.858353 [info ] [MainThread]: Profile pactravel_dbt_project written to /Users/macbookairm2/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m19:43:27.861942 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 65.16707, "process_in_blocks": "0", "process_kernel_time": 0.178744, "process_mem_max_rss": "109232128", "process_out_blocks": "0", "process_user_time": 0.934225}
[0m19:43:27.863098 [debug] [MainThread]: Command `dbt init` succeeded at 19:43:27.862875 after 65.17 seconds
[0m19:43:27.863731 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1029faa20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106fe1280>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1070960f0>]}
[0m19:43:27.864364 [debug] [MainThread]: Flushing usage events
[0m19:43:29.531678 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m19:43:37.175835 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105e50f50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1076925a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107692150>]}


============================== 19:43:37.177230 | 97ac4574-4d56-4206-8c7c-37a9ab2a888c ==============================
[0m19:43:37.177230 [info ] [MainThread]: Running with dbt=1.10.1
[0m19:43:37.177569 [debug] [MainThread]: running dbt with arguments {'indirect_selection': 'eager', 'send_anonymous_usage_stats': 'True', 'version_check': 'True', 'quiet': 'False', 'empty': 'None', 'fail_fast': 'False', 'log_format': 'default', 'use_experimental_parser': 'False', 'target_path': 'None', 'warn_error': 'None', 'no_print': 'None', 'cache_selected_only': 'False', 'use_colors': 'True', 'debug': 'False', 'printer_width': '80', 'profiles_dir': '/Users/macbookairm2/.dbt', 'introspect': 'True', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'invocation_command': 'dbt debug', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'log_path': 'logs'}
[0m19:43:37.182699 [info ] [MainThread]: dbt version: 1.10.1
[0m19:43:37.182906 [info ] [MainThread]: python version: 3.12.2
[0m19:43:37.183068 [info ] [MainThread]: python path: /opt/anaconda3/bin/python
[0m19:43:37.183221 [info ] [MainThread]: os info: macOS-13.4.1-arm64-arm-64bit
[0m19:43:37.208085 [info ] [MainThread]: Using profiles dir at /Users/macbookairm2/.dbt
[0m19:43:37.208301 [info ] [MainThread]: Using profiles.yml file at /Users/macbookairm2/.dbt/profiles.yml
[0m19:43:37.208470 [info ] [MainThread]: Using dbt_project.yml file at /Users/macbookairm2/Documents/Github/Pacmann---Data-Storage_-Warehouse--Mart---Lake/mentoring 3/pactravel-dataset-main/dbt_project.yml
[0m19:43:37.208696 [info ] [MainThread]: adapter type: postgres
[0m19:43:37.208843 [info ] [MainThread]: adapter version: 1.9.0
[0m19:43:37.208989 [info ] [MainThread]: Configuration:
[0m19:43:37.209124 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m19:43:37.209256 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m19:43:37.209388 [info ] [MainThread]: Required dependencies:
[0m19:43:37.209546 [debug] [MainThread]: Executing "git --help"
[0m19:43:37.217993 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m19:43:37.218429 [debug] [MainThread]: STDERR: "b''"
[0m19:43:37.218591 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m19:43:37.218782 [info ] [MainThread]: Connection:
[0m19:43:37.218950 [info ] [MainThread]:   host: localhost
[0m19:43:37.219101 [info ] [MainThread]:   port: 5441
[0m19:43:37.219239 [info ] [MainThread]:   user: postgres
[0m19:43:37.219370 [info ] [MainThread]:   database: pactravel_dbt_dwh
[0m19:43:37.219501 [info ] [MainThread]:   schema: dwh
[0m19:43:37.219631 [info ] [MainThread]:   connect_timeout: 10
[0m19:43:37.219760 [info ] [MainThread]:   role: None
[0m19:43:37.219889 [info ] [MainThread]:   search_path: None
[0m19:43:37.220017 [info ] [MainThread]:   keepalives_idle: 0
[0m19:43:37.220145 [info ] [MainThread]:   sslmode: None
[0m19:43:37.220272 [info ] [MainThread]:   sslcert: None
[0m19:43:37.220399 [info ] [MainThread]:   sslkey: None
[0m19:43:37.220525 [info ] [MainThread]:   sslrootcert: None
[0m19:43:37.220652 [info ] [MainThread]:   application_name: dbt
[0m19:43:37.220776 [info ] [MainThread]:   retries: 1
[0m19:43:37.221037 [info ] [MainThread]: Registered adapter: postgres=1.9.0
[0m19:43:37.276669 [debug] [MainThread]: Acquiring new postgres connection 'debug'
[0m19:43:37.327012 [debug] [MainThread]: Using postgres connection "debug"
[0m19:43:37.327214 [debug] [MainThread]: On debug: select 1 as id
[0m19:43:37.327359 [debug] [MainThread]: Opening a new connection, currently in state init
[0m19:43:37.328515 [debug] [MainThread]: Postgres adapter: Got a retryable error when attempting to open a postgres connection.
1 attempts remaining. Retrying in 0 seconds.
Error:
could not connect to server: Connection refused
	Is the server running on host "localhost" (127.0.0.1) and accepting
	TCP/IP connections on port 5441?
could not connect to server: Connection refused
	Is the server running on host "localhost" (::1) and accepting
	TCP/IP connections on port 5441?

[0m19:43:37.329143 [debug] [MainThread]: Postgres adapter: Error running SQL: select 1 as id
[0m19:43:37.329286 [debug] [MainThread]: Postgres adapter: Rolling back transaction.
[0m19:43:37.329427 [debug] [MainThread]: On debug: No close available on handle
[0m19:43:37.329576 [info ] [MainThread]:   Connection test: [[31mERROR[0m]

[0m19:43:37.329772 [info ] [MainThread]: [31m2 checks failed:[0m
[0m19:43:37.329912 [info ] [MainThread]: Project loading failed for the following reason:
 project path </Users/macbookairm2/Documents/Github/Pacmann---Data-Storage_-Warehouse--Mart---Lake/mentoring 3/pactravel-dataset-main/dbt_project.yml> not found

[0m19:43:37.330067 [info ] [MainThread]: dbt was unable to connect to the specified database.
The database returned the following error:

  >Database Error
  could not connect to server: Connection refused
  	Is the server running on host "localhost" (127.0.0.1) and accepting
  	TCP/IP connections on port 5441?
  could not connect to server: Connection refused
  	Is the server running on host "localhost" (::1) and accepting
  	TCP/IP connections on port 5441?
  

Check your database credentials and try again. For more information, visit:
https://docs.getdbt.com/docs/configure-your-profile


[0m19:43:37.330854 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.18641704, "process_in_blocks": "0", "process_kernel_time": 0.151216, "process_mem_max_rss": "113197056", "process_out_blocks": "0", "process_user_time": 0.748198}
[0m19:43:37.331100 [debug] [MainThread]: Command `dbt debug` failed at 19:43:37.331051 after 0.19 seconds
[0m19:43:37.331253 [debug] [MainThread]: Connection 'debug' was properly closed.
[0m19:43:37.331415 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107531010>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107c96e70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1076fa120>]}
[0m19:43:37.331608 [debug] [MainThread]: Flushing usage events
[0m19:43:38.561259 [debug] [MainThread]: An error was encountered while trying to flush usage events
